### DISCUSSION TITLE
I've developed a website which has a built in dark mode. Darkreader malforms the site, how do I make it say "using dark mode"?

### DISCUSSION THREAD

**Discussion by Maelstromeous** [darkreader/darkreader#13564] [Q&A] [answered]
I have a website (https://satisfactory-factories.app) that already uses a dark mode as part of it's core styling. Darkreader however malforms it:

<img width="1728" alt="Screenshot 2024-11-27 at 00 54 01" src="https://github.com/user-attachments/assets/defa6ec4-0c2b-48a8-a28e-8f4d083fbe82">

For some users it's really bad:

<img width="1728" alt="Screenshot 2024-11-27 at 00 54 01" src="https://private-user-images.githubusercontent.com/1776058/389210180-9382d7a3-f13e-42df-b640-585d7c2402d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI2NjkyMTgsIm5iZiI6MTczMjY2ODkxOCwicGF0aCI6Ii8xNzc2MDU4LzM4OTIxMDE4MC05MzgyZDdhMy1mMTNlLTQyZGYtYjY0MC01ODVkN2MyNDAyZDgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjdUMDA1NTE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDFmNzkyZjk3MzU1ZGM5ZTc4ZGI3ZDU1NDk5OWIzYmFmNjI3MzkwZTNkNWYxNjc5MzgzNjMzOWEyOTg4YzNhNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.KyxQ6GW5DIn7iMj6Ndk6oTKaYTuJvHa0VqUvQx12h2M">

It should look like this:

<img width="1728" alt="Screenshot 2024-11-27 at 00 54 18" src="https://github.com/user-attachments/assets/9180ed42-5f41-4cd2-a8b9-0e7522b7bac2">

Are there any ways for me to get darkreader to acknowledge the website already has a dark theme and automatically disable itself?

**Comment by Myshor** [comment, marked as answer]
I see you already found a solition, made PR and I merged it. 😉

**Reply by Maelstromeous** [reply]
Thanks! How exactly is the change applied? Presumably via extension updates?

**Reply by Myshor** [reply]
Exactly. With new version there will be updated dark-sites.config including added URL.

**Reply by Maelstromeous** [reply]
Thanks for the prompt answer and PR merge!
